packages/augur-sdk/src/state/db/MarketDB.ts

```diff
@@ -65,10 +67,19 @@ export class MarketDB extends DerivedDB {
       'MarketMigrated': this.processMarketMigrated,
     };
 
-    this.augur.events.subscribe('DB:updated:ZeroXOrders', (orderEvents) => this.syncOrderBooks([orderEvents.market]));
-    this.augur.events.subscribe('DB:get:ZeroXOrders', (markets) => this.syncOrderBooks(markets));
     this.augur.events.subscribe(SubscriptionEventName.NewBlock, this.processNewBlock);
     this.augur.events.subscribe(SubscriptionEventName.TimestampSet, this.processTimestampSet);
+
+    // Don't call this interval during tests
+    if (process.env.NODE_ENV !== 'test') {
+      if (!liquidityCheckInterval) {
+        // recalc liquidity every 3mins
+        const THREE_MINS_IN_MS = 180000;
+        liquidityCheckInterval = setInterval(() => {
+          this.syncOrderBooks([]);
```

Comment:
This is syncing every single market every 3 minutes. I think we should also still be listening to those events and marking which markets need to have their liquidity recalculated. Then when this gets called pass in those market ids.

Reply:
@nuevoalex ok that makes more sense, easy enough 👍